packages/app-bridge/src/react/useAssetBulkDownload.ts

```diff
@@ -0,0 +1,69 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
This hook and the tests are basically just a copy of `useBulkDownload`, which can be removed later on